src/main/java/org/primeframework/mvc/security/Encryptor.java

```diff
@@ -23,14 +23,16 @@
 public interface Encryptor {
   /**
    * Decrypt a set of bytes.
+   * <p>
+   * This will attempt to decrypt with GCM and then CBC if that fails.
```

Comment:
This should be a comment on `DefaultEncryptor` which is the implementation. The interface only defines a contract that says all implementations must be able to encrpt and decrypt things.